guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -4,6 +4,7 @@
 Use this procedure to import Kickstart repositories for {RHEL}{nbsp}{el-major-version}.
 
 .Procedure
+ifndef::orcharhino[]
```

Comment:
I do wonder if it wouldn't be better if you do provide more detailed instructions for RHEL customers, but I'll leave that to you.

Reply:
Yes, I will add something after introducing the `red_hat_enterprise_linux` attribute later on. For now, This is good enough for all different OS.

context: in downstream, we replace `{RHEL}` with all supported OS for managed hosts.